pkg/api/authz/authz.go

```diff
@@ -204,6 +204,11 @@ var (
 		types.GroupPowerUser: {
 			"GET /api/users",
 			"GET /api/users/{user_id}",
+			"GET /api/mcp-audit-logs",
+			"GET /api/mcp-audit-logs/filter-options/{filter}",
+			"GET /api/mcp-audit-logs/{mcp_id}",
+			"GET /api/mcp-stats",
+			"GET /api/mcp-stats/{mcp_id}",
```

Comment:
TODO: make sure that the PUs and PUPs can't use the `mcp_id` parameter to get info for servers outside of their workspace.

Reply:
This is working as expected.